quickwit-cli/src/index.rs

```diff
@@ -568,9 +570,18 @@ pub async fn ingest_docs_cli(args: IngestDocsArgs) -> anyhow::Result<()> {
         source_type,
         params,
     };
-    // Override index source config(s) with ad-hoc source config.
-    index_metadata.sources = vec![source_config];
 
+    run_checklist(vec![
```

Comment:
A few lines above, we've already done:
```rust
let metastore_uri_resolver = MetastoreUriResolver::default();
let metastore = metastore_uri_resolver.resolve(&args.metastore_uri).await?;
let mut index_metadata = metastore.index_metadata(&args.index_id).await?;
```

So, performing the connectivity check here is pointless.